.github/workflows/base-ci.yml

```diff
@@ -29,20 +29,29 @@ jobs:
     container: baoproject/bao:latest
     steps:
       - uses: actions/checkout@v2
```

Comment:
Just a small note: we must update the checkout action to v4 as v2 is deprecated. But this can be addressed in a later commit that does this for all workflows.